.github/workflows/hello_world_multiplatform.yaml

```diff
@@ -0,0 +1,78 @@
+name: Hello World (Multiplatform)
+
+on:
+  push:
+    branches:
+      - main
+      - v*-branch
+      - collab-*
+  pull_request:
+    branches:
+      - main
+      - v*-branch
+      - collab-*
+    paths:
+      - 'scripts/build/**'
+      - 'scripts/requirements*.txt'
+      - '.github/workflows/hello_world_multiplatform.yaml'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event_name }}-${{ github.head_ref || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-22.04, macos-12, windows-2022]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          path: zephyr
+          fetch-depth: 0
+
+      - name: Rebase
+        if: github.event_name == 'pull_request'
+        env:
+          BASE_REF: ${{ github.base_ref }}
+          PR_HEAD: ${{ github.event.pull_request.head.sha }}
+        working-directory: zephyr
+        shell: bash
+        run: |
+          git config --global user.email "actions@zephyrproject.org"
+          git config --global user.name "Github Actions"
+          git rebase origin/${BASE_REF}
+          git log --graph --oneline HEAD...${PR_HEAD}
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.11
+
+      - name: Setup Zephyr project
+        uses: zephyrproject-rtos/action-zephyr-setup@v1
+        with:
+          app-path: zephyr
+          toolchains: all
+
+      - name: Build firmware
+        working-directory: zephyr
+        shell: bash
+        run: |
+          if [ "${{ runner.os }}" = "macOS" ]; then
+            EXTRA_TWISTER_FLAGS="-P native_sim --build-only"
+          elif [ "${{ runner.os }}" = "Windows" ]; then
+            EXTRA_TWISTER_FLAGS="-P native_sim --short-build-path -O/tmp/twister-out"
+          fi
+          west twister --force-color --inline-logs -T samples/hello_world -v $EXTRA_TWISTER_FLAGS
```

Comment:
Still not seeing any coloured outputs in spite of `--force-color`:
https://github.com/zephyrproject-rtos/zephyr/actions/runs/7695949550/job/20969896275?pr=64802#step:6:31

I suspect this is because of the west wrapper (west itself would have already called colorama init before twister, which calls colorama init with strip disabled). Let's just directly call `./scripts/twister` as do the rest of the CI workflows.

Reply:
oky, let's see how this goes...